src/datalad_installer.py

```diff
@@ -2085,6 +2085,9 @@ def install_git_annex_dmg(
     dmgpath: Union[str, os.PathLike], manager: DataladInstaller
 ) -> Path:
     """Install git-annex from a DMG file at ``dmgpath``"""
+    if platform.machine() == "arm64":
+        log.info("M1 Mac detected; installing Rosetta")
+        runcmd("/usr/sbin/softwareupdate", "--install-rosetta", "--agree-to-license")
```

Comment:
is it possible to *quickly* check if rosetta is installed?

<details>
<summary>I have tried various softwareupdate options but found none which would list rosetta</summary> 

```shell
administrator@26696 ~ % /usr/sbin/softwareupdate --list
Software Update Tool

Finding available software
2022-04-19 11:05:05.637 softwareupdate[28580:1176211] XType: com.apple.fonts is not accessible.
2022-04-19 11:05:05.637 softwareupdate[28580:1176211] XType: XTFontStaticRegistry is enabled.
Software Update found the following new or updated software:
* Label: Command Line Tools for Xcode-13.3
	Title: Command Line Tools for Xcode, Version: 13.3, Size: 718145K, Recommended: YES, 
* Label: macOS Monterey 12.3.1-21E258
	Title: macOS Monterey 12.3.1, Version: 12.3.1, Size: 4303546K, Recommended: YES, Action: restart, 
administrator@26696 ~ % /usr/sbin/softwareupdate --list-full-installer
Finding available software
2022-04-19 11:05:22.774 softwareupdate[28591:1176554] XType: com.apple.fonts is not accessible.
2022-04-19 11:05:22.775 softwareupdate[28591:1176554] XType: XTFontStaticRegistry is enabled.
Software Update found the following full installers:
* Title: macOS Monterey, Version: 12.3.1, Size: 12225260436K, Build: 21E258
* Title: macOS Monterey, Version: 12.3, Size: 12222979494K, Build: 21E230
* Title: macOS Monterey, Version: 12.2.1, Size: 12155426708K, Build: 21D62
* Title: macOS Monterey, Version: 12.2, Size: 12156325205K, Build: 21D49
* Title: macOS Monterey, Version: 12.1, Size: 12157035487K, Build: 21C52
* Title: macOS Monterey, Version: 12.0.1, Size: 12128428704K, Build: 21A559
* Title: macOS Big Sur, Version: 11.6.5, Size: 12412317772K, Build: 20G527
* Title: macOS Big Sur, Version: 11.6.4, Size: 12439328867K, Build: 20G417
* Title: macOS Big Sur, Version: 11.6.3, Size: 12435122667K, Build: 20G415
* Title: macOS Big Sur, Version: 11.6.2, Size: 12433351292K, Build: 20G314
* Title: macOS Big Sur, Version: 11.6.1, Size: 12428472512K, Build: 20G224
* Title: macOS Big Sur, Version: 11.5.2, Size: 12440916552K, Build: 20G95
administrator@26696 ~ % /usr/sbin/softwareupdate --history            
Display Name                                       Version    Date                  
------------                                       -------    ----                  
Command Line Tools for Xcode                       13.2       02/01/2022, 06:51:26 
```
</details>

if we don't find any -- I guess it is ok indeed to just run it again

Reply:
@yarikoptic I'm not sure how to check for that.